RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate ChangePasscodeView to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Go to `Security and Privacy`
- Enable the `Unlock with Passcode`, if is your first time unlocking with the passcode you will insert the passcode normally
- Then will appear a button `Change Passcode`, that will pop up a modal to you insert the passcode
- All functionalities should work properly.

## Screenshots

## Types of changes




## Checklist



## Further comments
